app/models/local_transaction_edition.rb

```diff
@@ -49,4 +55,12 @@ def build_clone(target_class = nil)
     end
     new_edition
   end
+
+private
+
+  def merge_embedded_devolved_administration_availability_errors
+    [scotland_availability, wales_availability, northern_ireland_availability].each do |administration|
+      errors.merge! administration.errors
+    end
+  end
```

Comment:
> I wondered if something like this might needed for JS errors (perhaps keys like wales_availability_alternative_url => []) but when I tried that it seemed like it's rather broken across the app.

Yeah, I found this too.  Not the only broken thing either - the JS version (Ajax) simply doesn't display any messages except a sort of catch-all `We had some problems saving. Please check the form below.` in a bar along the bottom of the page - nothing seems to be highlighted and I can't see any meaningful error messages.  Would be good to get an approach for this PR as I suspect a proper fix needs to be done as another story.

Reply:
The way things are structured currently makes a nice `wales_availability: { alternative_url: "can't be blank" }` type hash difficult to achieve.